src/vm/gdbjit.cpp

```diff
@@ -1029,6 +1030,79 @@ void TypeMember::DumpStaticDebugInfo(char* ptr, int& offset)
     offset += ptrSize + 2;
 }
 
+void FunctionMember::MangleName(char *buf, int &buf_offset, const char *name)
```

Comment:
Since we generate a separate ELF for each compiled method, there is no problem with identical mangled names. We just need a valid mangled name so that lldb can see the the method as function (with prologue, epilogue etc...).